### PR TITLE
Use "being draggable"  block instead of defining a custom block 

### PR DESCRIPTION
**Pull request by bethanyaconnor**
The `being draggable` block is in the GamelabJr pool but this level was defining it as a custom block. Removing it from the "custom block" field doesn't remove the block from the toolbox but does allow the translations for this block to be used.

Production:
Screenshot 2020-02-21 at 4 21 59 PM

Local with change:
Screenshot 2020-02-21 at 4 21 55 PM

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-390)

## Testing story

I created a simple project locally that tests stopping and starting the `being draggable` behavior and it works as expected.
Screenshot 2020-02-24 at 10 59 55 AM

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by ajpal** [comment]
Does this change need to be made on these other levels, too? 
image

File: dashboard/config/scripts/levels/New Sprite Lab Project.level [hunk 1]
```diff
@@ -71,7 +71,7 @@
...
-  "level_concept_difficulty": {
```
Comment (ajpal) [inline dashboard/config/scripts/levels/New Sprite Lab Project.level]: intentional?

File: dashboard/config/scripts/levels/New Sprite Lab Project.level [hunk 2]
```diff
@@ -487,4 +487,4 @@
       </xml>
     </toolbox_blocks>
   </blocks>
-</GamelabJr>
\ No newline at end of file
+</GamelabJr>
```
Comment (ajpal) [inline dashboard/config/scripts/levels/New Sprite Lab Project.level]: intentional?

Comment (bethanyaconnor) [reply dashboard/config/scripts/levels/New Sprite Lab Project.level]: Nope -- that's my editor. I'll revert that

**Comment by bethanyaconnor** [comment]
Yeah those will also all need to be fixed but I think I'd rather update them case-by-case as a lot of those look like test levels.

@dju90 to weigh in

**Review by Hamms — APPROVED** [review]
Can you confirm that the definitions of the two blocks are identical? If so, LGTM!

**Comment by bethanyaconnor** [comment]
Custom blocks from the `New Spritelab Project` level:
``` 
{
    "func": "draggable",
    "blockText": "being draggable",
    "returnType": "Behavior",
    "color": [
      136,
      ".84",
      ".80"
    ]
  }
```

Definition of the `being draggable` block in the GamelabJr block pool:
```
{
  "func": "draggable",
  "blockText": "being draggable",
  "returnType": "Behavior",
  "color": [
    136,
    ".84",
    ".80"
  ]
}
```

**Comment by bethanyaconnor** [comment]
After talking with @ajpal we realized that `custom_blocks` is actually completely redundant on this level.

These are the custom blocks that are defined:
Screenshot 2020-02-24 at 2 46 50 PM

And this is the behavior toolbox:
Screenshot 2020-02-24 at 3 42 03 PM

The only custom block actually used is `being draggable` ... which we know isn't needed.

**Review by Hamms — APPROVED** [review]

**Pull request merged**